### PR TITLE
SSMD Recon Corvette

### PR DESCRIPTION
**Pull request by TheDanseMacabre**
This PR adds a SSMD recon corvette to the badlands, and fixes some minor issues with other ships. It also adds a Visegradi flag and a Solarian military IPC uniform (and fixes some odd shading while I was at it)

### CODE REVIEW

**Comment by TheDanseMacabre** [comment]
!wip

**Comment by Dreamykass** [comment]
Please, I beg you, more airlocks, or even a proper docking port.

Currently, as in the mapdiffs, this has only two single-tile airlocks, that are effectively in a single spot (aft, near the hangar). This makes anything EVA related super miserable. You can't cycle with a piece of machinery or a canister or similar things (that you can't climb on top) if it's a single tile airlock. Not to mention interacting with other third parties, trying to visit each other, or board in combat.

**Comment by DreamySkrell** [comment]
Please, I beg you, more airlocks, or even a proper docking port.

Currently, as in the mapdiffs, this has only two single-tile airlocks, that are effectively in a single spot (aft, near the hangar). This makes anything EVA related super miserable. You can't cycle with a piece of machinery or a canister or similar things (that you can't climb on top) if it's a single tile airlock. Not to mention interacting with other third parties, trying to visit each other, or board in combat.

**Comment by TheDanseMacabre** [comment]
!review

File: maps/away/ships/sol_ssmd/ssmd_ship.dm [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+	suffix = "ships/sol_ssmd/ssmd_ship.dmm"
+	sectors = list(SECTOR_BADLANDS)
+	spawn_weight = 1
+	spawn_cost = 1
+	id = "ssmd_corvette"
+	shuttles_to_initialise = list(/datum/shuttle/autodock/overmap/ssmd_shuttle)
+
+/decl/submap_archetype/ssmd_corvette
+	map = "SSMD Corvette"
+	descriptor = "A long-range reconnaissance corvette design in use by the Solarian Navy, the Uhlan-class is a relatively costly and somewhat uncommon ship to be seen in the Alliance’s fleets, and is typically reserved for more elite (or at least better-equipped and trained) units. Designed to operate alone or as part of a small task force with minimal support in unfriendly space, it is most commonly seen assigned to probing, surveillance, harassment, and strike operations. \
```
Comment (SleepyGemmy) [inline maps/away/ships/sol_ssmd/ssmd_ship.dm:14]:
```suggestion
	descriptor = "A long-range reconnaissance corvette design in use by the Solarian Navy, the Uhlan-class is a relatively costly and somewhat uncommon ship to be seen in the Alliance's fleets, and is typically reserved for more elite (or at least better equipped and trained) units. Designed to operate alone or as part of a small task force with minimal support in unfriendly space, it is most commonly seen assigned to probing, surveillance, harassment, and strike operations. \
```

File: maps/away/ships/sol_ssmd/ssmd_ship.dm [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+	name = "In transit"
+	landmark_tag = "nav_transit_ssmd_corvette"
+	base_turf = /turf/space/transit/north
+
+//shuttle stuff
+/obj/effect/overmap/visitable/ship/landable/ssmd_shuttle
+	name = "SSMD Shuttle"
+	class = "SSMDV"
+	designation = "Vizsla"
+	desc = "A modestly-sized shuttle design used by the Solarian armed forces, the Destrier is well-armored but somewhat slow, and was explicitly designed to be as survivable as possible for operations during combat. Notably features a fast-deployment exosuit catapult."
```
Comment (SleepyGemmy) [inline maps/away/ships/sol_ssmd/ssmd_ship.dm:135]:
```suggestion
	desc = "A modestly sized shuttle design used by the Solarian armed forces, the Destrier is well-armored but somewhat slow, and was explicitly designed to be as survivable as possible for operations during combat. Notably features a fast-deployment exosuit catapult."
```

File: maps/away/ships/sol_ssmd/ssmd_ship_ghostroles.dm [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+
+	backpack_contents = list(/obj/item/storage/box/survival = 1, /obj/item/melee/energy/sword/knife/sol = 1)
+
+/datum/outfit/admin/ssmd_ipc/get_id_access()
+	return list(access_sol_ships, access_external_airlocks)
+
+//items
+
+/obj/item/card/id/ssmd_ship
+	name = "ssmd patrol ship id"
```
Comment (SleepyGemmy) [inline maps/away/ships/sol_ssmd/ssmd_ship_ghostroles.dm:165]:
```suggestion
	name = "\improper SSMD patrol ship identification card"
```

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
consider a PR description.

**Review by SleepyGemmy — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**